### DISCUSSION TITLE
AzureAD groups automatic mapped to grafana teams

### DISCUSSION THREAD

**Discussion by mkarci** [grafana/grafana#48962] [General]
Hi all, 

The team sync unfortunately works only when you have manually created a team. Within the team setting you are able to add external groups sync, so each user of this group will be sync automatically to this team.

Eventually what you want is that also the the groups defined within AzureAD mapped automatically to the teams within in Grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
